joomla/joomla-cms
# Add info for enabling the workflow

Pull Request for Issue #34741  .

### Summary of Changes
Add some information how to enable the workflow in J4.


### Testing Instructions
Code review 
or
apply patch, install sample data in a fresh installation and check the workflow text.


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request
![grafik](https://user-images.githubusercontent.com/1035262/133596054-4add01a5-8cc4-41ad-a2b1-dd6ddfbb81a0.png)



### Documentation Changes Required
